LimitedDailyIncome.lua

```diff
@@ -4,8 +4,14 @@ LimitedDailyIncome.sales = {}
 LimitedDailyIncome.wasPlayerOnline = {}
 LimitedDailyIncome.salesLimit = {}
 
-LimitedDailyIncome.STANDARD_LIMIT = 500000
-LimitedDailyIncome.INCREASE_LIMIT_FACTOR = 250000
+-- Daily Limit
+LimitedDailyIncome.STANDARD_LIMIT = 450000
```

Comment:
```suggestion
LimitedDailyIncome.STANDARD_LIMIT = 500000
```
I would stick with exactly half a million as defaultValue. But it is just personal preference